services/movie.py

```diff
@@ -28,13 +33,16 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
 ) -> Movie:
```

Comment:
According to the checklist, you should use double quotes in your code. Single quotes are used here.